examples/weather/.scalafix.conf

```diff
@@ -0,0 +1,10 @@
+rules = [
+  OrganizeImports
+]
+
+OrganizeImports {
+  groupedImports                    = Merge
+  coalesceToWildcardImportThreshold = 6
+  groups                            = ["re:javax?\\.", "cats.", "scala.", "org.tessellation.", "com.my.nft.", "*"]
```

Comment:
might want to add the namespace you used here so that the imports format into groups